.pre-commit-config.yaml

```diff
@@ -438,6 +438,17 @@ repos:
         files: ^airflow/www/.*\.(css|scss|sass)$
         # Keep dependency versions in sync w/ airflow/www/package.json
         additional_dependencies: ['stylelint@13.3.1', 'stylelint-config-standard@20.0.0']
+      - id: version-sync
+        name: Version sync
+        files: ^airflow/version.py$|setup.py
+        entry: ./scripts/ci/pre_commit/pre_commit_sync_version.sh
+        language: system
```

Comment:
@ashb I had to add this one as there was a bug in doc script that gave ma a really hard time. The __init__.py file generated for doc building time was not removed (because it used rmtree and rmrtree did not remove the file. 

I fixed it and added pre-commit to make sure if this will not occur in the future.

Reply:
Is this needed anymore with Kamil's doc change?

Reply:
I think yes. There were a few cases in various configurations where dynamic loading of versions from setup.py did not work. I thin the number of times It did not work for me (in pre-commits mainly but also when installing packages and sources or when people will be installing virtualenv this might cause various problems. I believe hard-coding the version and making sure it is synchronized at the commit time removes the potential failure paths and clears the path so that it  "just works".